pkgs/os-specific/darwin/apple-sdk/default.nix

```diff
@@ -106,6 +106,8 @@ let
           cp -R "${sdk.out}/Library/Frameworks/$name.framework/Versions/$current/Headers" .
         fi
 
+        ln -s -L "/System/Library/Frameworks/$nested_path/Versions/$current/$name"
```

Comment:
This is bringing back the impure symlinks to system frameworks that no longer exist on Big Sur, and in combination with  "apple-sdk: only try to copy tbd files if they exist", this is almost completely reverting Big Sur support.

Reply:
After fixing the wrong stat this is no longer necessary.